### PR TITLE
M5 g 144 move announcement bubble

### PR DESCRIPTION
**Pull request by nbhatia823**
This PR moves the AnnouncementBubble that currently lives in Launchpad to Dewey! And it wraps message bodies in `<Linkify>` so that way messages will have content hyperlinked

Screen Shot 2021-04-09 at 4 25 10 PM
.
**Overview:**

**Screenshots/GIFs:**

**Testing:**

- [ ] Unit tests
- Manual tests:
  - [ ] Chrome
  - [ ] Safari
  - [ ] IE11

**Roll Out:**

- Before merging:
  - [x] Updated docs
  - [x] Bumped version in `package.json`
    - Breaking change?
      - If it is a beta component run `npm version minor`
      - If the component is not in beta run `npm version major`
    - New component or backward-compatible component feature change? Run `npm version minor`
    - Only changing documentation? All good. Skip this step.
  - After creating a new component, make sure to add it to the Components List in `ComponentsView.jsx`. To do so:
    - [ ] Add an entry in `ComponentsView.componentsToDisplay` using this template:
      ```
      {
        componentLink: "<COMPONENT LINK>",
        componentImg: "<COMPONENT LINK>.png",
        componentName: "<COMPONENT NAME>",
        componentImgAlt: "A <COMPONENT NAME> component",
      },
      ```
    - [ ] Add a screenshot of the component in `docs/assets/img` with the format `<COMPONENT LINK>.png`
- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)
  - [ ] Posted in #eng if I made a breaking change to a beta component

### CODE REVIEW

File: src/AnnouncementBubble/DeletedAnnouncementBubble.less [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+@media only screen and (max-width: @breakpointM) {
+  .DeletedAnnouncementBubble--icon {
+    margin: 0.625rem 0.5rem 0.875rem 0.75rem;
+    .text--large();
+  }
+
+  .DeletedAnnouncementBubble--text {
+    margin: 0.75rem 0.75rem 0.75rem 0;
```
Comment (jonjonray) [inline src/AnnouncementBubble/DeletedAnnouncementBubble.less:27]: so much precision

File: src/MessagingBubble/MessagingBubble.less [hunk 2]
```diff
@@ -4,7 +4,7 @@
 .MessagingBubble--Message {
   border-radius: 0.3125rem;
   font-size: 1rem;
-  max-width: 36.5rem;
+  max-width: 32.5rem;
```
Comment (jonjonray) [inline src/MessagingBubble/MessagingBubble.less:7]: your payment is acknowledged

File: src/AnnouncementBubble/QuotedAnnouncementBubble.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  // If the content is not expanded, limit the message length so that screen readers
+  // don't attempt to read the entire hidden message. This only works for plain-string children
+  let content = children;
+  if (!isExpanded && typeof content === "string") {
+    content = content.substring(0, 100);
+  }
+
+  const buttonText = isExpanded ? "Show less" : "Show more";
+
+  // TODO: Wrap in Linkify component when moved to Dewey similar to MessagingBubble
```
Comment (jonjonray) [inline src/AnnouncementBubble/QuotedAnnouncementBubble.tsx:46]: nit: can probably remove this TODO

**Review by jonjonray — APPROVED** [review]
LGTM

File: src/AnnouncementBubble/QuotedAnnouncementBubble.tsx (hunk 3 above)
Comment (nbhatia823) [reply src/AnnouncementBubble/QuotedAnnouncementBubble.tsx:46]: Truth

**Comment by nbhatia823** [comment]
Thanks for reviewing!

**Pull request merged**